db/seeds/dev/colors.js

```diff
@@ -0,0 +1,39 @@
+const colors = require('../../../colors');
+const users = require('../../../users');
+const projects = require('../../../projects');
+
+const hexColors = colors.map(color => ({
+  hex_code: color
+}));
+
+const theUsers = users.map(user => ({
+  email: user.email,
+  password: user.password,
+  name: user.name
+}));
+
+const theProjects = projects.map(user => ({
+  user_id: project.user_id,
+  color_1_id: project.color_1_id,
+  color_2_id: project.color_2_id,
+  color_3_id: project.color_3_id,
+  color_4_id: project.color_4_id,
+  color_5_id: project.color_5_id,
+}));
+
+exports.seed = async function(knex) {
+  try {
+    await knex('colors').del();
+    await knex.raw('ALTER SEQUENCE colors_id_seq RESTART WITH 1');
+    await knex('users').del();
+    await knex.raw('ALTER SEQUENCE users_id_seq RESTART WITH 1');
+    await knex('projects').del();
+    await knex.raw('ALTER SEQUENCE users_id_seq RESTART WITH 1');
+    await knex('projects').insert(theUsers);
+    await knex('users').insert(theUsers);
+
```

Comment:
This is all new to me. Is this effectively making sure that the IDs start at 1 each time a seed is run?

Reply:
I ran into some complications after a migration rollback. I re-migrated, re-seeded, and saw that the starting ID's were off - almost as if there were phantom values remaining from the previous seed data in table. I tried truncating the tables and reseeding, yet still ran into starting IDs that were non-zero.- which caused issues with matching ID's in my GET endpoint testing.

This code forces the column IDs to start at 1